package.json

```diff
@@ -14,10 +14,11 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.15",
-    "@docusaurus/preset-classic": "^2.0.0-beta.15",
+    "@docusaurus/core": "^2.0.0-beta.17",
+    "@docusaurus/preset-classic": "^2.0.0-beta.17",
     "@mdx-js/react": "^1.6.21",
-    "@svgr/webpack": "^5.5.0",
+    "@svgr/webpack": "^6.2.1",
+    "acorn": "^8.7.0",
```

Comment:
what is "acorn"? If it's not needed, we can remove it. Also, we always wanna avoid changing the dependency versions in order to avoid merge conflicts and other potential problems, so please change them back to what they previously were, and do the same for the `package-lock.json` file. This can easily be done from the source control tab in vscode.